src/source/PeerConnection/SessionDescription.c

```diff
@@ -582,6 +610,42 @@ STATUS populateSessionDescriptionDataChannel(PKvsPeerConnection pKvsPeerConnecti
     return retStatus;
 }
 
+BOOL isPresentInRemote(PKvsRtpTransceiver pKvsRtpTransceiver, PSessionDescription pRemoteSessionDescription) {
```

Comment:
This API assumes that all of the parameters have been sanitized, could you please confirm.

Reply:
confirmed